plugins/lookup/passwordstore.py

```diff
@@ -125,6 +145,11 @@
 - name: Return the entire password file content
   ansible.builtin.set_fact:
     passfilecontent: "{{ lookup('community.general.passwordstore', 'example/test returnall=true')}}"
+
+ansible.cfg: >
```

Comment:
```suggestion
ansible.cfg: |
```
`>` removes line breaks. That will result in a single long line, which is clearly not what you want.

Reply:
Thanks. That was just copy and pasted from [here](https://github.com/ansible-collections/community.general/blob/00cab64f7abb687cc2abf600843477e3f4484308/plugins/callback/diy.py#L605) without thinking.